packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,137 @@
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
+
+#### Nimble Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `error-visible` - boolean
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design also includes an `error` state. This will be controlled with the boolean attribute `error-visible`, which has a default attribute value of "". When needed, "error-visible" would be added to the accordion attributes, changing the color of the accordion item border color to red. This will be implemented in the accordion styling through conditional css styles based on `error-visible`'s value (ex. :host([error-visible]) will have styling for when `error-visible` is true).
+
+### Angular integration
+
+An Angular directive will be created for this component. The component will not have form association, so a `ControlValueAccessor` will not be created.
+
+### Blazor integration
+
+A blazor wrapper will be created for the component.
+
+### Additional requirements
+
+-   _User interaction: Do the FAST component's behaviors match the visual design spec? When they differ, which approach is preferable and why?_
+    -   No Additional Requirements
+-   _Styling: Does FAST provide APIs to achieve the styling in the visual design spec?_
+    -   No Additional Requirements
+-   _Testing: Is FAST's coverage sufficient? Should we write any tests beyond Chromatic visual tests?_
+    -   No Additional Requirements
+-   _Documentation: Any requirements besides standard Storybook docs and updating the Example Client App demo?_
+    -   No Additional Requirements
+-   _Tooling: Any new tools, updates to tools, code generation, etc?_
+    -   No Additional Requirements
+-   _Accessibility: keyboard navigation/focus, form input, use with assistive technology, etc._
+    -   No Additional Requirements
+-   _Globalization: special RTL handling, swapping of icons/visuals, localization, etc._
+    -   No Additional Requirements
+-   _Performance: does the FAST component meet Nimble's performance requirements?_
+    -   No Additional Requirements
+-   _Security: Any requirements for security?_
+    -   No Additional Requirements
+
+---
+
+## Open Issues
+
+Should we only have arrow icons for open / closed indicators on the accordion, or should other visuals like +/- be implemented as well?
```

Comment:
I'm curious where the idea came from to add other visuals? I don't see them in the design spec. If it's just something we think someone might want later, I'd vote to remove it from the spec for now and add it on-demand if someone ever asks for it.